pod-operation/src/components/lim_temperature.rs

```diff
@@ -6,6 +6,19 @@ use ads1x1x::{Ads1x1x, DynamicOneShot, SlaveAddr};
 use nb::block;
 use rppal::i2c::I2c;
 
+const DIVIDER_RESISTANCE: f32 = 22000.0;
+const VCC: f32 = 5.0;
```

Comment:
Nitpick: V<sub>cc</sub> isn't entirely accurate since the circuit doesn't have a common collector. Perhaps consider `V_IN`?